src/Qir/Runtime/public/CoreTypes.hpp

```diff
@@ -26,10 +26,10 @@
   dereferenced in client's code.
 ==============================================================================*/
 class QUBIT;
-typedef QUBIT* Qubit;
+typedef QUBIT* Qubit;   // TODO: Replace with `typedef uint64_t QubitId`. Remove all the `GetQubitId()`, `QUBIT`.
 
 class RESULT;
-typedef RESULT* Result;
+typedef RESULT* Result; // TODO: Replace with `typedef uintXX_t Result`, where XX is 8|16|32|64. Remove all the `RESULT`.

```

Comment:
Do you need to take care of these ToDo's before merging?

Reply:
Resolved in a different discussion earlier.